synapse/cli.py

```
@@ -381,8 +382,8 @@ def cleanup(signum, frame):
 
     print(f"\x1b[32m[Synapse]\x1b[0m Starting {profile} on port {port}")
     print(f"\x1b[32m[Synapse]\x1b[0m Submit sequence: {repr(submit_seq)}")
-    print(f"\x1b[32m[Synapse]\x1b[0m Use @Agent to send messages to other agents")
-    print(f"\x1b[32m[Synapse]\x1b[0m Use @Agent --response 'message' to get response here")
+    print(f"\x1b[32m[Synapse]\x1b[0m Use @Agent 'message' to send (response expected by default)")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)